bears/js/JSComplexityBear.py

```diff
@@ -3,24 +3,17 @@
 
 from coalib.bearlib.abstractions.Linter import linter
 from coalib.bears.requirements.NpmRequirement import NpmRequirement
+from coalib.misc.Compatability import JSONDecodeError
```

Comment:
I see, my spelling was wrong in the coala's repo as well, and you've changed it :+1: